k8s/main.ts

```diff
@@ -11,9 +11,7 @@ export class MyChart extends PennLabsChart {
     const backendImage = 'pennlabs/penn-courses-backend';
     const secret = 'penn-courses';
 
-    new RedisApplication(this, 'redis', { deployment: { 
-	    image: 'redis/redis-stack-server',
-	    tag: '6.2.6-v6' } });
+    new RedisApplication(this, 'redis/redis-stack', { deployment: { tag: '418539118ee8' } });
```

Comment:
@rohangpta

Reply:
Absolutely, this change is more centered around using `redis-stack`. Vanilla redis doesn't ship with the json and search modules by default. 

Pinning the tag to that version is more of a config mgmt practice where we're deliberately avoiding using a latest version (in case it might have some breaking backwards incompatible changes) -- this is less of a focus though.